### PR TITLE
Update docs to reflect CHUNK_TYPE configuration and TS.INFO's debug f…

### LINKED ISSUE #571: Update docs with `CHUNK_TYPE` and `TS.INFO` new flag

### PR DESCRIPTION
**Pull request by danni-m**
…lag.

Also fix `bytesPerSample` to respond with a float instead of an integer.
Fix #571

### CODE REVIEW

File: src/module.c [hunk 1]
```diff
@@ -384,7 +384,7 @@ int TSDB_info(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
             RedisModule_ReplyWithSimpleString(ctx, "size");
             RedisModule_ReplyWithLongLong(ctx, chunkSize);
             RedisModule_ReplyWithSimpleString(ctx, "bytesPerSample");
-            RedisModule_ReplyWithDouble(ctx, chunkSize / numOfSamples);
+            RedisModule_ReplyWithDouble(ctx, (float)chunkSize / numOfSamples);
```
Comment (gkorland) [inline src/module.c:387]: @danni-m I guess this commit is not related (and removes a warning), does it fix a bug?

Comment (danni-m) [reply src/module.c:387]: @gkorland its not totally related, but I found this while making sure the docs align with the actual output.

Comment (gkorland) [reply src/module.c:387]: How come it didn't fail any test? should we change any test to cover this change?

**Review by gkorland — APPROVED** [review]
